src/universal/utils/transit.js

```diff
@@ -20,7 +20,6 @@
 
 import transit from 'transit-immutable-js';
 import { serializeError } from 'serialize-error';
-import { TimeoutError } from './createTimeoutFetch';
```

Comment:
So `transit.js` no longer needs a TimeoutError? Is that because it is handled in prerender.js

Reply:
So TimeoutError is still an instance of Error so should still be handled, createTimeoutFetch also uses abort controller rather and falling back to the timeout error if abort controller fails, this should never happen especially on the server.

Reply:
So TimeoutError is still an instance of Error so should still be handled, createTimeoutFetch also uses abort controller rather and falling back to the timeout error if abort controller fails, this should never happen especially on the server.